drivers/sensors/adxl372.c

```diff
@@ -716,10 +720,12 @@ static void adxl372_dvr_exchange(FAR void *instance_handle,
                                  FAR const void *txbuffer,
                                  FAR void *rxbuffer, size_t nwords)
 {
-  FAR struct adxl372_dev_s *priv = (FAR struct adxl372_dev_s *)instance_handle;
+  FAR struct adxl372_dev_s *priv = instance_handle;
```

Comment:
Why is the implicit cast removed here?

Reply:
To fix the long length warning.

Reply:
@Ouss4 I short instance_handle to instance, please review again.